src/components/draggable/draggable-container.component.js

```diff
@@ -6,26 +6,34 @@ import styledSystemPropTypes from "@styled-system/prop-types";
 
 import { filterStyledSystemMarginProps } from "../../style/utils";
 import DraggableItem from "./draggable-item.component";
-import { StyledIcon, StyledDraggableContainer } from "./draggable-item.style";
+import {
```

Comment:
Instead of adding `renderAsUl` and `renderAsLi` props couldn't you just pass `as` props like that? :
```javascript
<DraggableContainer as="ul">
  <DraggableItem key="1" id={1} as="li">
    simple content
  </DraggableItem>
  <DraggableItem key="2" id={2} as="li">
    simple content
  </DraggableItem>
</DraggableContainer>
```
All you'd have to do is just to ensure that `as` is passed straight to the wanted StyledComponent